### DISCUSSION TITLE
Probe temp changes without input

### DISCUSSION THREAD

**Discussion by tmfaulk** [artisan-roaster-scope/artisan#1067] [Q&A]
Thanks for the opportunity to ask for help.  I did not see this issue in search.
I've had a change in K style probe temps and to my knowledge I made no changes.

For the first 2 years on my 3k mill city I used Artisan with great results.  
For reference FC (3K) is generally 390F with K style probes.  2 years ago I moved
to a Mill City 20k with K style probes. I imported Artisan from the 3K to the computer
on the 20K.  For reference FC (20K) is generally 365F.  I continued to roast the
similar previous coffees while building profiles on the 20K

I roasted hundreds of lbs on Thursday (no issues) and came to roast Monday and the Artisan icon
on the desktop was gone and when I opened the normal profiles, the background
was different.  It didn't strike me as an issue so I roasted 4 smaller batches and
I thought the roasts looked light.  I roasted a batch that called to go into SC.  It was
very light.  SC generally on the 20K is 403F but on this batch it didn't reach SC until 415F.
These are beans I roast weekly so very consistent. BTW in retrospect all 4 batches 
were definitely lighter in roast level.

It seems like it reverted to 3K type profile readings.  I didn't see anything changed in 
the probe settings but the phase temps did represent 3K settings.

Any suggestions in this matter would be greatly appreciated.
tmfaulk

**Comment by roasterdave** [comment]
>came to roast Monday and the Artisan icon on the desktop was gone

You don't mention what OS platform you are running, however this behavior is outside of Artisan.  Did your computer perform any sort of "maintenance" over the weekend?

**Comment by tmfaulk** [comment]
Dave
Thanks for the reply.
Both computers are PCs with
Windows 10.
Neither are connected to the internet
so it would not have run an update.
I think this is what you are asking.
TF

**Comment by roasterdave** [comment]
Hard for me to guess at what happened external to Artisan.  What I recommend is make two settings files, one for the 3K and one for the 20K.  Use the menu Help>> Save settings and save with an appropriate file name.  If this ever happens again you can recover quickly using Help> Load settings.

**Comment by tmfaulk** [comment]
Thanks. That's kinda what I was going to do at this moment and reinstall
artisan.  I should not lose profiles if sure in a folder, correct?
TF

**Reply by roasterdave** [reply]
Neither Artisan nor its installer will delete your profiles or settings.  Given the odd behavior of that PC you might want to make a backup of those files.  It's good practice anyway to protect against any sort of data loss.

**Comment by tmfaulk** [comment]
Thanks again for the advice
Will do.
TF

**Comment by tmfaulk** [comment]
Dave
One more question if I can..
My version is 2.4.0 build 1
Should I install the latest update?
I have windows 10pro on the roaster pc
Thank you

**Comment by roasterdave** [comment]
Yes, you should upgrade to the current release, which is v2.8.2, then keep current with the latest releases.  It is easy to check for new releases from within Artisan using Help>> Check for Updates.

**Comment by tmfaulk** [comment]
Dave thank you for your response.
I appreciate your time.
Tommy Faulk
RuedeCafe' of Acadiana
337-654-8769
